### ISSUE TITLE
Motherload mining info panel for "Mining" vs "Not Mining" should have a valid idle timer

### ISSUE THREAD

**Issue by amp4045** [runelite/runelite#4953]
**Describe the bug**
It appears the information panel (where it displays pay-dirt mined, pay-dir/hr, etc.) should contain a valid idle timer. Currently while still mining, the player has small stalls between animations, this causes the information panel to update with "NOT mining", when in reality the user IS still mining, it's just stalled slightly due to animation.

**To Reproduce**
1. Go to motherload mine with the plugin enabled
2. Begin mining and watch the information panel bounce between mining and not mining while the ore vain is still validly accessible...and your character is in fact still mining.

**Expected behavior**
Perhaps an idle timer can be allocated to this plugin such that it will only update to "NOT mining" after a few seconds of being idle. 

**Screenshots**
https://imgur.com/5pEQgSr

**Environment (please complete the following information):**
 - OS: Windows 
 - RuneLite version: 1.4.12
-  Oldschool revision: Rev 172
 - Launcher version: UNKNOWN

**Additional context**
This problem was discovered using a rune pickaxe. I'm not sure if other pickaxes cause the same issue, or at the same rate (I'd assume yes, though).

**Comment by amp4045** [comment]
apologies! I tried searching for this issue specifically via motherload before posting.

**Comment by deathbeam** [comment]
yeah, it was under MLM i think: https://github.com/runelite/runelite/issues/1377
